.github/workflows/tests.yml

```diff
@@ -236,12 +216,7 @@ jobs:
             postgres:${{ matrix.job.postgres-version }}
 
       - name: Install Rust
-        # There don't seem to be versioned releases of this action per se: for each rust
-        # version there is a branch which gets constantly rebased on top of master.
-        # We pin to a specific commit for paranoia's sake.
-        uses: dtolnay/rust-toolchain@fc3253060d0c959bea12a59f10f8391454a0b02d
-        with:
-            toolchain: 1.58.1
+        uses: dtolnay/rust-toolchain@1.58.1
```

Comment:
Does this mean this is now versioned and the comment is obsolete?

Reply:
The comment is still accurate: the underlying logic/mechanism of the action isn't associated to a release of the action.

However:

> # We pin to a specific commit for paranoia's sake.

I've decided that the paranoia doesn't justify this getting updated by dependabot semifrequently. (We [Synapse] talked about this during our Monday meeting, which I think you missed---see the notes at the bottom of the minutes.)

Reply:
I've amended the description with words along these lines.